### PR TITLE
SupportBundles: Add support bundle documentation

### PR DESCRIPTION
**Pull request by Jguer**
https://github.com/grafana/website/pull/11458 (img)

- Add Support bundle feature description

To be merged for release

### CODE REVIEW

File: docs/sources/troubleshooting/support-bundles/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - grafana
+  - troubleshooting
+  - support
+  - bundles
+title: Send a support bundle to Grafana Labs support
+menutitle: Send a support bundle to support
+weight: 100
+---
+
+# Send a panel to Grafana Labs support
```
Comment (Jguer) [inline docs/sources/troubleshooting/support-bundles/index.md:13]:
```suggestion
# Send a support bundle to Grafana Labs support
```

File: docs/sources/troubleshooting/support-bundles/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+b. Attach the archive (tar.gz) file to a support ticket that you send to Grafana Labs Technical Support.
+
+## Available support bundle components
+
+- **Usage statistics**: Usage statistic for grafana instance
+- **User information**: A list of users of the grafana instance
+- **Database and Migration information**: Database information and migration log
+- **Plugin information**: Plugin information for grafana instance
+- **Basic information**: Basic information about the grafana instance (Version, mem use information, goroutines…)
```
Comment (IevaVasiljeva) [inline docs/sources/troubleshooting/support-bundles/index.md:48]:
```suggestion
- **Basic information**: Basic information about the grafana instance (version, memory usage etc)
```

**Review by IevaVasiljeva — APPROVED** [review]
LGTM!

File: docs/sources/troubleshooting/support-bundles/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  - grafana
+  - troubleshooting
+  - support
+  - bundles
+title: Send a support bundle to Grafana Labs support
+menutitle: Send a support bundle to support
+weight: 200
+---
+
+# Send a support bundle to Grafana Labs support
```
Comment (chri2547) [inline docs/sources/troubleshooting/support-bundles/index.md:13]: @Jguer , is this available for all users, including OSS? If not, then we need to call out that this feature is for enterprise customers only.

File: docs/sources/troubleshooting/support-bundles/index.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+When you encounter problems with your Grafana instance, you can send us a support bundle that contains information about your Grafana instance, including:
+
+- Grafana version
+- Installed plugins
+- Grafana configuration
+- Deployed database information and migrations
+
+## Available support bundle components
+
```
Comment (chri2547) [inline docs/sources/troubleshooting/support-bundles/index.md:23]: @Jguer , I moved this section up so it's visible before the steps. We need an intro sentence after the heading but before the list. What does available support bundle components mean?

File: docs/sources/troubleshooting/support-bundles/index.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+1. Select the components that you want to include in the support bundle.
+
+1. Click **Create**.
+
+1. After the support bundle is ready, click **Download**.
+
+   Grafana downloads the support bundle to an archive (tar.gz) file.
+
+1. Attach the archive (tar.gz) file to a support ticket that you send to Grafana Labs Technical Support.
```
Comment (chri2547) [inline docs/sources/troubleshooting/support-bundles/index.md:52]: @Jguer , what is the email address they should send the bundle to?

**Review by chri2547 — COMMENTED** [review]
@Jguer , added a few questions. Also committed some changes.

File: docs/sources/troubleshooting/support-bundles/index.md (hunk 3 above)
Comment (Jguer) [reply docs/sources/troubleshooting/support-bundles/index.md:13]: Creating a support bundle is available for all users.

Sending these support bundles to GL support is reserved for enterprise

File: docs/sources/troubleshooting/support-bundles/index.md (hunk 5 above)
Comment (Jguer) [reply docs/sources/troubleshooting/support-bundles/index.md:52]: Normally not an email but a support ticket (this line was copied from the Dashboard support section)

File: docs/sources/troubleshooting/support-bundles/index.md (hunk 4 above)
Comment (Jguer) [reply docs/sources/troubleshooting/support-bundles/index.md:23]: Each component is a file inside of the support bundle.

Users can choose to include different information in the support bundle (for example: only settings and SAML information or only users list)

Comment (chri2547) [reply docs/sources/troubleshooting/support-bundles/index.md:23]: got it, ok, adding a small suggestion.

Comment (chri2547) [inline docs/sources/troubleshooting/support-bundles/index.md:23]:
```suggestion

A support bundle can include any of the following components:
```

File: docs/sources/troubleshooting/support-bundles/index.md (hunk 5 above)
Comment (chri2547) [reply docs/sources/troubleshooting/support-bundles/index.md:52]: As long as users know how to do this, I'm fine with it. The dashboard instructions are also not clear on this point. I had asked about this and didn't hear a response, so we went with what we had.

File: docs/sources/troubleshooting/support-bundles/index.md (hunk 3 above)
Comment (chri2547) [reply docs/sources/troubleshooting/support-bundles/index.md:13]: Ok, thank you.

**Review by chri2547 — APPROVED** [review]
One suggestion, LGTM! Thank you!

**Pull request merged**